### ISSUE TITLE
Script does no longer work

### ISSUE THREAD

**Issue by ha1fdan** [FlagX/ha-ledvance-tuya-resync-localkey#19]
python .\print-local-keys.py
Please put your Tuya/Ledvance username: xxxxxxxx@gmail.com
Please put your Tuya/Ledvance password:
---------------------------
device name:    Smart Plug 1
device id:      bf94c09abf90xxxxxxx
local key:      hGwPLA)T_Z%F>{PL
---------------------------
device name:    Smart Plug 2
device id:      bf9772ae0daxxxxxxx
local key:      '!}{g82vEr^E`4&f
---------------------------
Press Enter to continue...

**Comment by ha1fdan** [comment]
I can control my smart plugs via the app

**Comment by FlagX** [comment]
@ha1fdan, you printed the local keys here. So I would say the script has worked as intended?

**Comment by ha1fdan** [comment]
Yes but the keys are encrypted in some way, and homeassistant won't accept the keys as the script print them.

**Comment by FlagX** [comment]
They look pretty similar to what I would get for my lamps (16 chars, alphanumeric with special characters) and for me it still works. Any error message or hints in the logs of home assistant?

**Comment by mafen** [comment]
works just fine for me

**Comment by ha1fdan** [comment]
Okay, let me try again tomorrow.

**Comment by ha1fdan** [comment]
Hey, you know what? It was my home assistant that was bad! Not your program, thanks so much👍
